### ISSUE TITLE
Safari: Missing visual focus and cannot tab to open links

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#5034]
<img width="1172" alt="Screen Shot 2019-10-16 at 1 46 28 PM" src="https://user-images.githubusercontent.com/45409515/66944548-63c2ba00-f01b-11e9-87bf-4270c3ca55c7.png">

**Comment by KateChuen** [comment]
Tested on dev3. It works after enabling in safari (#5041)
